acq_stat_reports/config.py

```diff
@@ -0,0 +1,21 @@
+class Config:
+    # "Top-level data directory",
+    data_dir = "."
```

Comment:
Maybe `web_dir_default` or `plot_dir_default`? I think it is currently being used in that way.

Reply:
Same comment as above. I agree that this needs changing, but I already started #9 and I don't want to start making changes that cause conflict with the PR downstream.